SU2_CFD/include/solvers/CFVMFlowSolverBase.inl

```diff
@@ -2558,7 +2584,12 @@ void CFVMFlowSolverBase<V, FlowRegime>::Friction_Forces(const CGeometry* geometr
       /*--- Compute y+ and non-dimensional velocity ---*/
 
       FrictionVel = sqrt(fabs(WallShearStress[iMarker][iVertex]) / Density);
-      YPlus[iMarker][iVertex] = WallDistMod * FrictionVel / (Viscosity / Density);
+
+      /* --- in case of wall functions, we have computed YPlus in the turbulence class --- */
+      if (!wallfunctions)  
+        YPlus[iMarker][iVertex] = WallDistMod * FrictionVel / (Viscosity / Density);
+
+
```

Comment:
Is there a standard oneliner that I and everybody else can run on all files to autoformat everything?

Reply:
No we have not made it that drastic yet. If you install clang-format on your machine you can do `clang-format "file you want" > "result"` then replace the original with the formatted.
This was done to CFVMFlowSolverBase when the file was created, but it has not been done for a while now, so probably it would introduce changes that are not related with what you are working on... But you can try it and see what it looks like.